content/messaging/customer-chat-api/v3.2/index.mdx

```diff
@@ -1167,6 +1167,92 @@ curl -X POST \
 
 </Section>
 
+<Section>
+<Text>
+
+### List Threads
+
+It returns threads that the current Agent has access to in a given chat.
+
+#### Specifics
+
+|                        |                                                                  |
+| ---------------------- | ---------------------------------------------------------------- |
+| **Method URL**         | `https://api.livechatinc.com/v3.2/agent/action/list_threads` |
+| **Required scopes**    | `chats--all:ro` `chats--access:ro`                               |
+| **RTM API equivalent** | [`list_threads`](rtm-reference/#list-threads)            |
+| **Webhook**            | -                                                                |
+
+#### Request
+
+| Parameter          | Required | Data type | Notes                                                                                      |
+| ------------------ | -------- | --------- | ------------------------------------------------------------------------------------------ |
+| `chat_id`          | Yes      | `string`  |                                                                                            |
+| `sort_order` 		 | No       | `string`  | Possible values: `asc` - oldest threads first and `desc` - newest threads first (default). |
```

Comment:
q: will returned threads be sorted or should I sort them on my side?

Reply:
they should be sorted